### PR TITLE
feat: add linux/arm64 support

### PR DESCRIPTION
**Pull request by glyphtech-chrisa**
plus `go mod tidy` required fix.

### CODE REVIEW

**Comment by glyphtech-chrisa** [comment]
Context: Deploying ArgoCD via Helm to arm64 hosts and fails pulling the image for this. I've tested the arm64 image some locally and appears to work in K8s 1.23.

**Comment by glyphtech-chrisa** [comment]
Hi Team, please review/approve when you get a chance: @crenshaw-dev @rishabh625 @jgwest @ishitasequeira @mmerrill3.

File: go.mod
Comment (rishabh625) [inline go.mod:100]: This fix is already present in master branch, how come diff shows it as new addition

Comment (crenshaw-dev) [reply go.mod:100]: I think merging master would remove it from the diff. But it shouldn't cause any problems as-is.

Comment (glyphtech-chrisa) [reply go.mod:100]: I fixed this and put out the PR before that was merged into master I believe.

**Review by wtam2018 — APPROVED** [review]
LGTM
thanks @glyphtech-chrisa !

**Pull request merged**
